androidApp/src/main/java/dev/johnoreilly/confetti/wear/WearSettingsSync.kt

```diff
@@ -115,7 +118,21 @@ class WearSettingsSync(
         }
     }
 
-    suspend fun installOnWearNode(nodeId: String) {
+    // public non-suspend function to be called from button callbacks, etc...
+    fun installOnWearNode(nodeId: String) {
```

Comment:
This is pretty quick, I wouldn't necessarily worry about the Job. Running it twice is unlikely and probably safe.

Reply:
Probably a style thing, but what about this just being a suspend function, and using a higher level scope when calling it from a VM?

Reply:
But can I then use the `coroutineScope` when calling it from compose? Like

```
    val scope = rememberCoroutineScope()
    fun installOnWear() {
        wearSettingNodeState.filter { !it.isAppInstalled }.forEach {
            scope.launch {
                wearSettingSync.installOnWearNode(it.id)
            }
        }
    }

```

Because I think that was what we were a bit worried about that that scope would be easily cancelled and canceling the call 🤔

Reply:
I was suggesting pulling it up to the app ViewModel.

Reply:
FWIW, `AppViewModel` is really `InitialLoadingViewModel` in the model where there is a 1:1 relationship between VM and screens. I plead guilty of very bad naming there. 

If we introduce an app-level VM, it'd be parallel to `AppState` I guess and used from the ConfettiApp composable.
That would break the 1:1 screen:VM relationship I was operating under but that'd work for me too, we can be pretty flexible if we don't start creating a VM for each single composable.

Reply:
Probably better we head for a common understanding and approach. Feels free to ship this and dismiss my suggestions. Nothing blocking.

Reply:
👍 Merging this PR, let's follow up in separate PRs for style/aligment.